lighthouse-cli/bin.ts

```diff
@@ -241,8 +242,15 @@ function showProtocolTimeoutError() {
   process.exit(_PROTOCOL_TIMEOUT_EXIT_CODE);
 }
 
+function showPageLoadError() {
+  console.error('Unable to load the page.');
+  process.exit(_PAGE_LOAD_ERROR_EXIT_CODE);
```

Comment:
interesting. Any reason to have a special error code (vs using `_RUNTIME_ERROR_CODE`)? No big deal if so, we added `_PROTOCOL_TIMEOUT_EXIT_CODE` specifically because we needed to detect that case for Travis retries